ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/tap-aggregation.j2

```diff
@@ -0,0 +1,42 @@
+{# eos - tap-aggregation #}
+{% if tap_aggregation is arista.avd.defined %}
+!
+tap aggregation
+{%     if tap_aggregation.mode.exclusive is arista.avd.defined %}
+{%         if tap_aggregation.mode.exclusive.enabled is arista.avd.defined(true) %}
+{%             set mode_cli = "mode exclusive" %}
+{%         endif %}
+{%         if tap_aggregation.mode.exclusive.profile is arista.avd.defined %}
+{%             set mode_cli = mode_cli ~ " profile " ~ tap_aggregation.mode.exclusive.profile %}
+{%         endif %}
+   {{ mode_cli }}
+{%         for interface in tap_aggregation.mode.exclusive.no_errdisable | arista.avd.natural_sort %}
+   mode exclusive no-errdisable {{ interface }}
+{%         endfor %}
+{%     endif %}
+{%     if tap_aggregation.encapsulation_dot1br_strip is arista.avd.defined(true) %}
+   encapsulation dot1br strip
+{%     endif %}
+{%     if tap_aggregation.encapsulation_vn_tag_strip is arista.avd.defined(true) %}
+   encapsulation vn-tag strip
+{%     endif %}
+{%     if tap_aggregation.protocol_lldp_trap is arista.avd.defined(true) %}
+   protocol lldp trap
+{%     endif %}
+{%     if tap_aggregation.truncation_size is arista.avd.defined %}
+   truncation size {{ tap_aggregation.truncation_size }}
+{%     endif %}
+{%     if tap_aggregation.mac.timestamp.replace_source_mac is arista.avd.defined(true) %}
+   mac timestamp replace source-mac
+{%     elif tap_aggregation.mac.timestamp.header.format is arista.avd.defined %}
+   mac timestamp header format {{ tap_aggregation.mac.timestamp.header.format }}
+{%     endif %}
+{%     if tap_aggregation.mac.timestamp.header.eth_type is arista.avd.defined %}
+   mac timestamp header eth-type {{ tap_aggregation.mac.timestamp.header.eth_type }}
+{%     endif %}
```

Comment:
All of this should be in one block, under ```.header```

Reply:
thanks @tgodaA i've created the section